<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectLogWidget</class>
 <widget class="QWidget" name="ProjectLogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Log Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_stop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Log settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QTraitLineEdit" name="le_project">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Female ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Male ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTraitLineEdit" name="le_maleid">
        <property name="toolTip">
         <string>Base filename, log.csv will generate files log000000.csv, log000001.csv, etc..</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Project</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTraitLineEdit" name="le_femaleid"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTraitLineEdit</class>
   <extends>QLineEdit</extends>
   <header>madernpytools/qtgui/traitlet_widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
